account_asset_depr_line_cancel/static/description/icon.svg

```diff
@@ -0,0 +1,391 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why? The file is the source file for the image, so it should remain. Please don't remove it.